vitrina/settings.py

```diff
@@ -135,12 +135,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.mysql',
+        'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'adp-dev',
-        'USER': 'root',
-        'PASSWORD': 'r00t',
-        'HOST': '192.168.2.60',
-        'PORT': '3306'
+        'USER': 'adp',
+        'PASSWORD': 'secret',
+        'HOST': '127.0.0.1',
+        'PORT': '5432',
```

Comment:
Could you please change that to [environ.Env.db_url](https://django-environ.readthedocs.io/en/latest/types.html#environ-env-db-url)?